packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -44,9 +44,11 @@ import { createPreStartStrategy } from './preStartRum'
 import type { StartRum, StartRumResult } from './startRum'
 
 export type RumPublicApi = ReturnType<typeof makeRumPublicApi>
-
+export interface StartStrategyOptions {
+  forceStart: boolean
```

Comment:
I have no strong opinion, but I guess `DD_RUM.startSessionReplayRecording({ force: true })` would b clear, since the public method state that it's about starting the recording

Reply:
Done ✔️